supreme_court_predictions/models/logistic_regression.py

```diff
@@ -1,48 +1,97 @@
-        self.logistic_regression()
+        self.total_utterances = pd.read_pickle(
+            self.local_path + "case_aggregations.p"
+        )
+        self.advocate_utterances = pd.read_pickle(
+            self.local_path + "advocate_aggregations.p"
+        )
+        self.adversary_utterances = pd.read_pickle(
+            self.local_path + "adversary_aggregations.p"
+        )
+        self.judge_utterances = pd.read_pickle(
+            self.local_path + "judge_aggregations.p"
+        )
+
+        self.run()
 
-    def logistic_regression(self):
+    @staticmethod
+    def create(df):
         """
-        TODO: Need document string
+        Creates and runs a logistic regression on the given dataframe of
+        utterance data.
+
+        :param df: DataFrame containing utterance data
+
+        :return float: Accuracy score of the logistic regression model
         """
-        vectorizer = CountVectorizer()
-        vectorize_document = self.utterances_df.loc[:, "tokens"].apply(" ".join)
+        vectorizer = CountVectorizer(analyzer="word", max_features=5000)
+        vectorize_document = df.loc[:, "tokens"].apply(" ".join)
         print("Creating bag of words")
         bag_of_words_x = vectorizer.fit_transform(vectorize_document)
 
-        # TODO: after Chay's merge of dataframe, this can be updated.
-        bag_of_words_y = np.random.randint(0, 2, len(vectorize_document))
+        bag_of_words_y = df.loc[:, "win_side"]
 
         X_train, X_test, y_train, y_test = train_test_split(
-            bag_of_words_x, bag_of_words_y, test_size=0.25, random_state=123
+            bag_of_words_x,
+            bag_of_words_y,
+            test_size=0.20,
+            random_state=123,
+            stratify=bag_of_words_y,
         )
 
-        print("Starting the Logistic Regression on utterances")
-        regressor = skLR()
+        print("Starting the Logistic Regression")
+        regressor = skLR(max_iter=1000)
 
         # Fit the classifier on the training data
         regressor.fit(X_train, y_train)
         y_pred = regressor.predict(X_test)
 
-        print("Prediction done: ", y_pred)
+        return accuracy_score(y_true=y_test, y_pred=y_pred)
+
+    def run(self):
+        """
+        Runs the create function on each type of aggregated utterance.
+        """
+
+        dfs = [
+            ("total_utterances", self.total_utterances),
+            ("judge_utterances", self.judge_utterances),
+            ("advocate_utterances", self.advocate_utterances),
+            ("adversary_utterances", self.adversary_utterances),
+        ]
+
+        for df_name, df in dfs:
+            try:
+                print("------------------------------------------")
+                print(f"Running regression on {df_name}...")
+                acc = self.create(df)
+                print(f"Accuracy score: {acc}")
+                print("------------------------------------------")
+            except ValueError:
```

Comment:
The linter complained if it did not have an error assigned so I went with `ValueError` based off some searching around.